### PR TITLE
form load happening before form component load

### PR DESCRIPTION
**Pull request by ann-aot**
In order to maintain the reactiveness, removed the `v-if ` and added `watch`

### CODE REVIEW

**Review by henryoforeh-aot — APPROVED** [review]

**Review by guru-aot — APPROVED** [review]
LGTM

File: sources/packages/web/src/components/common/StudentApplication.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <formio
+    v-if="Object.keys(initialData).length > 0"
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/common/StudentApplication.vue:3]: Please change for a computed property 😉

File: sources/packages/web/src/components/common/StudentApplication.vue [hunk 2]
```diff
@@ -91,75 +91,43 @@ export default {
...
-      applicationWizard.on("wizardPageSelected", (page: any, index: number) => {
-        isFirstPage.value = index === 0;
-        isLastPage.value = applicationWizard.isLastPage();
-        // Event to set isInFirstPage, current page and isInLastPage to parent
-        context.emit(
-          "pageChanged",
-          isFirstPage.value,
-          applicationWizard.page,
-          isLastPage.value,
+    const formDependencies = async () => {
```
Comment (andrewsignori-aot) [inline sources/packages/web/src/components/common/StudentApplication.vue:94]: Can we call it `loadFormDependencies`? Just a suggestion 😉

**Review by andrewsignori-aot — APPROVED** [review]
Looks good @ann-aot. Thanks for checking a way to make the form work based on the reactiveness of the _initialData_ 👍

**Review by dheepak-aot — APPROVED** [review]
LGTM. Nice way to put the fix 👍

**Pull request merged**